src/backends/onnxruntime.c

```diff
@@ -115,7 +111,13 @@ OrtValue* RAI_OrtValueFromTensor(RAI_Tensor* t, RAI_Error *error) {
 }
 
 RAI_Tensor* RAI_TensorCreateFromOrtValue(OrtValue* v, RAI_Error *error) {
-  if (!OrtIsTensor(v)) {
+  OrtStatus* status = NULL;
+
+  int is_tensor;
+  status = OrtIsTensor(v, &is_tensor);
+  if (status != NULL) goto error;
+
+  if (!is_tensor) {
```

Comment:
I was wondering whether we should free ret, shape, and strides at the error handler to avoid leaks.

Reply:
Done